_infra/helm/mock-eq/templates/deployment.yaml

```diff
@@ -21,36 +25,23 @@ spec:
         image: "{{ .Values.image.devRepo }}/{{ .Chart.Name }}:{{ .Values.image.tag }}"
         {{- end}}
         imagePullPolicy: Always
+        volumeMounts:
+        - name: google-cloud-key
+          mountPath: /var/secrets/google
         ports:
         - name: http-server
-          containerPort: 8080
+          containerPort: {{ .Values.container.port }}
         resources:
-          limits:
-            cpu: "200m"
-            memory: "200Mi"
-          requests:
-            cpu: "100m"
-            memory: "100Mi"
+          {{ toYaml .Values.resources | nindent 12 }}
         env:
         - name: JSON_SECRET_KEYS
           valueFrom:
             secretKeyRef:
               name: mock-eq
               key: mock-eq
-      # readinessProbe:
```

Comment:
I think this stuff is actually needed to let k8s know how to ask the app if it's healthy/ready.  Can you add this stuff back?  In theory it should just work as it's very similar to others

Can you also add a very simple /info endpoint to this repo.  Just something that returns something like:
```python
{"status": "healthy"}
```
or something basic like that.